### PR TITLE
Add support for post widgets & widget subareas

### PR DESCRIPTION
**Pull request by valendesigns**

### CODE REVIEW

File: php/class-customize-snapshot.php [hunk 1]
```diff
@@ -296,9 +296,45 @@ public function values() {
...
 				return $setting['dirty'] === $dirty;
 			} );
+
+			$customize_widgets = $this->snapshot_manager->customize_manager->widgets;
+			$dirty_widgets = array();
+
+			// Find widgets in the dirty sidebars.
+			foreach ( $dirty_values as $setting_id => $setting ) {
+				if ( -1 !== strpos( $setting_id, 'sidebars_widgets' ) ) {
+					$dirty_widgets += (array) $setting['value'];
```
Comment (westonruter) [inline php/class-customize-snapshot.php]: Using `+` for array concatenation yields unexpected results. I think you actually mean `array_merge()` here? See for example: https://3v4l.org/50WYF

Comment (westonruter) [inline php/class-customize-snapshot.php]: Maybe:

``` php
$dirty_widgets = array_unique( array_merge( $dirty_widgets, (array) $setting['value'] ) );
```

Comment (valendesigns) [inline php/class-customize-snapshot.php]: array_merge would work correctly

**Comment by westonruter** [comment]
Closing this in favor of #9.

**Pull request closed without merging**